package.json

```diff
@@ -20,13 +20,15 @@
     "orm"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc && yarn build:strict",
```

Comment:
Why do we need another build command? Cannot we "strict" build the entire project?

Reply:
Because right now we have two tsconfig files ([explanation here](https://github.com/wovalle/fireorm/pull/174#discussion_r464701300)). The `build:strict` script will only type check the files that are currently in the `include` option. Note that the `buid:strict` **does not actually builds the project, it just type-checks the files** (this is because of the `noEmit` option). The other build (which is the old one) is the one that actually builds the project. The idea is that after the migration is finished we can remove the second `build:strict` command (in conjunction of the removal of the duplicate tsconfig). Is it clearer now?

Reply:
yeah, makes sense! Do you think we can rename the strict "tsconfig.json" to "tsconfig.strict.json" and move it to the root?

Reply:
It's possible, but I don't know how the editors will deal with the `tsconfig.strict.json` file (I believe it will ignore it and thus we will lose the strict type-chek showing up on the editor)

Reply:
@wovalle tested moving the tsconfig to the root and happened what I said above (the editor doesn't show the errors, running `build:strict` works though). So if we move the config we will lose the immediate feedback (even though I prefer the organization that moving to the root brings).WDYT we should do?

Reply:
I prefer a good developer experience. Eventually it'll be removed so lets leave it for now